Vizzuality/heco-invest
# Add map layers legends

This PR adds the map layers legends

## Acceptance criteria

The designs of the legend can be found [here](https://www.figma.com/file/3epSYh4KQZBCyh4Y5OzmHw/HeCo---Design-System?node-id=3656%3A12756), there is an example of style per type of data.

### HeCo priority landscapes
Title only with small square in #3d6c50

### Sub-basin boundaries
Title only with small square in ​#2485bc

### Protected areas
Title only with small square in ​#bafdd1 (70% opacity)

### Biodiversity intactness
Gradient: #f8ebff (label "Low"), #eccafc, #dfa4ff, #c26dfe, #9d36f7, #6d00e1, #3c00ab (label "High")[RW data object](https://api.resourcewatch.org/v1/dataset/1ddacdc5-b05e-4ba0-91e6-9b99c66f70b2/layer)

### Tree biomass density
Gradient: #895122 (label “0”), #957f4f, #9db38a, #39a401 (label "480 t Ha⁻¹")[RW data object](https://api.resourcewatch.org/v1/dataset/587b9951-54aa-4cd5-8929-8aacd5743e15/layer)

### Wetlands
Title only with small square in ​#0d9dc9 (60% opacity)

### Population density
Choropleth: #32095d (label "≤25"), #781c6d (label "≤100"), #ba3655 (label "≤1k"), #ed6825 (label "≤5k"), #fbb318 (label "≤10k"), #fcfea4 (label ">10k")[RW data object](https://api.resourcewatch.org/v1/dataset/f6b81da7-a846-45c1-88b2-dad95a26e523/layer)

### Endangered species and critical habitats
Title only with small square in #b2d26e[RW data object](https://api.resourcewatch.org/v1/dataset/5b5a21ac-0835-43fb-86b9-64b93d472e10/layer/4b5140a2-28ea-4707-8998-a9d1c32dbf53)

### Tree cover loss
Title only with small square in #dc6c9a[RW data object](https://api.resourcewatch.org/v1/dataset/b584954c-0d8d-40c6-859c-f3fdf3c2c5df/layer)

### Riverine flood risk
Choropleth: #990000 (label "Extremely high"), #ff1900 (label "High"), #ff9900 (label "Medium to high"), #ffe600 (label "Low to medium"), #ffff99 (label "Low"), #808080 (label "Arid & low water use"), #4e4e4e (label "No data")[RW data object](https://api.resourcewatch.org/v1/dataset/5152c286-53c1-4583-9519-816a6e41889d/layer)

### Indigenous Reserves
Title only with small square in ​#ffb780 (70% opacity)

## Testing instructions

- In discover/project page, enable and disable the map layers
- The legends should be displayed at the bottom right of the map
- Be sure that the legends can be enabled and disabled properly

## Tracking

[LET-1054](https://vizzuality.atlassian.net/browse/LET-1054)
